pkg/question/question.go

```diff
@@ -121,6 +121,13 @@ func AskQuestion(input *AskQuestionInput) string {
 
 		// No match at all
 		fmt.Println("Input invalid. Please try again.")
```

Comment:
Can we add a new unit test to cover this invalid input test case, so that we can capture this issue in the future? Something similar to https://github.com/awslabs/aws-simple-ec2-cli/blob/main/pkg/question/question_test.go#L83-L90, for example.

Reply:
Added unit test case for invalid inputs as per the discussion.